pkg/registry/authorization/selfsubjectaccessreview/rest.go

```diff
@@ -73,6 +76,11 @@ func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation
 	}
 
 	var authorizationAttributes authorizer.AttributesRecord
+	if len(userToCheck.GetExtra()[authorizationv1.ScopesKey]) > 0 {
```

Comment:
this would allow self-SARs to tokens aren't supposed to be used for that purpose